downloader/src/funTest/kotlin/GitRepoDownloadTest.kt

```diff
@@ -88,14 +66,23 @@ class GitRepoDownloadTest : StringSpec() {
 
             val actualSpdxFiles = spdxDir.listFiles(FileFilter { it.isDirectory }).map { it.name }.sorted()
 
+            val submodulesDir = File(outputDir, "submodules")
+            val expectedSubmodulesFiles = listOf(
```

Comment:
Should we call this (and the variable below) "Dirs" at the end as they only contain directories?

Reply:
I'd rather keep it consistent with the existing code and fix it together separately.